keras/layers/recurrent.py

```diff
@@ -366,5 +366,306 @@ def get_config(self):
             "inner_activation":self.inner_activation.__name__,
             "truncate_gradient":self.truncate_gradient,
             "return_sequences":self.return_sequences}
+
+
+
+class MutatedRNN_1(Layer):
```

Comment:
Definitely not a fan of the names (in terms of style: it's a mixture of camel case and snake case, in terms of UX it's not descriptive, as the space of all possible genetically evolved RNNs is infinite).